docs/reference/eql/syntax.asciidoc

```diff
@@ -54,6 +54,30 @@ For example, the following EQL query matches any documents with a
 any where network.protocol == "http"
 ----
 
+[discrete]
+[[eql-syntax-escape-identifiers]]
+==== Escape an event category or field name
+
+Event categories or field names containing non-alphanumeric characters must be
+escaped with enclosing backticks (+++`+++). Non-alphanumeric characters include
+underscores (`_`), dots (`.`), hyphens (`-`), or spaces.
```

Comment:
I believe underscores are valid. Also, if a field name starts with a number (i.e. `10x`, then it also needs to be wrapped in backticks)

Reply:
Thanks! I haven't tested this yet so I'll confirm.

Reply:
and `dots` are still fine for fields, as long as they are in the middle and not at the end. `process.name` is still valid

Reply:
Thanks for pointing this out @rw-access. Was able to confirm in my testing. I won't mention dots at all as ES doesn't allow the creation of fields starting or ending with a dot.